BDKSwiftExampleWallet/Extensions/BDK+Extensions/Outpoint+Extensions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Outpoint+Extensions.swift
+//  BDKSwiftExampleWallet
+//
+//  Created by Matthew Ramsden on 6/3/25.
+//
+
+import Foundation
+import BitcoinDevKit
+
+extension OutPoint: Hashable {
```

Comment:
Outpoint went to Record from object so lost its Hashable/Equatable in the process, so added this because of that